Assets/MRTK/SDK/Features/Utilities/Migration/Tools/MigrationTool.cs

```diff
@@ -46,6 +61,18 @@ public MigrationTool()
         /// </summary>
         public bool TryAddObjectForMigration(Type type, Object selectedObject)
         {
+            if (MigrationState == MigrationToolState.Migrating)
+            {
+                Debug.LogError("Objects cannot be added during migration process.");
+                return false;
+            }
+            else if (MigrationState == MigrationToolState.PostMigration)
+            {
+                Debug.Log("Cleaning list of processed objects.");
```

Comment:
Nit: Might be worth deleting this log, as it will show up in the unity log and be kinda cryptic (what is this list of processed objects?)